api/src/controllers/login.js

```diff
@@ -17,6 +17,15 @@ const translations = require('../translations');
 const template = require('../services/template');
 const rateLimitService = require('../services/rate-limit');
 const serverUtils = require('../server-utils');
+const { validatePassword } = require('@medic/user-management/src/users');
```

Comment:
Rather than linking to the internals of the user-management shared-lib treat it like all you can see is the main index.js file. If you just require `@medic/user-management` and update the index.js to export the `validatePassword` function then it's a cleaner interface.